### PR TITLE
Updated installation.md with respect to information about CVAT_HOST for local installation

### LINKED ISSUE #4013: Update installation guide for exporting CVAT_HOST env variable before docker compose command.
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [X] Searched [past issues](/issues)

### Expected Behaviour

The installation guide should mention about exporting `CVAT_HOST` variable so that CVAT can be access over the network. The current installation method only supports accessing CVAT using `http://localhost:8080`. This information about exporting `CVAT_HOST` variable can be derived from AWS-Deployment Guide 
[source here](https://github.com/openvinotoolkit/cvat/blob/develop/site/content/en/docs/administration/basics/AWS-Deployment-Guide.md)
[actual here](https://openvinotoolkit.github.io/cvat/docs/administration/basics/aws-deployment-guide/)

### Current Behaviour

After installation, CVAT cannot be accessed over the network, as CVAT_HOST variable exporting information is not provided.

### Possible Solution

Provide instructions in installation manual to export `CVAT_HOST` variable.

### Steps to Reproduce (for bugs)

1. [Actual Documentation Link](https://openvinotoolkit.github.io/cvat/docs/administration/basics/installation/)

### Context

Perform repeated installation and solve the problem through trial-n-error or else
after reading `docker-compose.yml` in detail.

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by anugrahasinha**
Resolves #4013 

### Motivation and context

The installation guide should mention about exporting CVAT_HOST variable so that CVAT can be access over the network. The current installation method only supports accessing CVAT using http://localhost:8080. This information about exporting CVAT_HOST variable can be derived from AWS-Deployment Guide
[source here](https://github.com/openvinotoolkit/cvat/blob/develop/site/content/en/docs/administration/basics/AWS-Deployment-Guide.md)
[actual here](https://openvinotoolkit.github.io/cvat/docs/administration/basics/aws-deployment-guide/)

### How has this been tested?

This is document level change, therefore the markdown has been checked for sanity.

### Checklist

- [X] I submit my changes into the `develop` branch
- [X] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [X] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [X] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [X] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@anugrahasinha , thanks for your contribution! Great job. I will merge the PR after all linter checks are passed.

File: site/content/en/docs/administration/basics/installation.md [hunk 1]
```diff
@@ -76,7 +76,12 @@ For access from China, read [sources for users from China](#sources-for-users-fr
   git clone https://github.com/opencv/cvat
   cd cvat
   ```
-
+- To access CVAT over a network or through a different system, export `CVAT_HOST` environment variable
```
Comment (azhavoro) [inline site/content/en/docs/administration/basics/installation.md:79]: From my point of view, it is better not to duplicate this, but add a link to the corresponding section of the FAQ page: https://openvinotoolkit.github.io/cvat/docs/faq/#how-to-change-default-cvat-hostname-or-port

Comment (nmanovic) [reply site/content/en/docs/administration/basics/installation.md:79]: FAQ and documentation has different purpose. Let's have the duplication for now. In the future we will add more details into documentation but FAQ should be short.

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Points of invisible shapes are visible in autobordering (<https://github.com/openvinotoolkit/cvat/pull/3931>)
 - Order of the label attributes in the object item details(<https://github.com/openvinotoolkit/cvat/pull/3945>)
 - Order of labels in tasks and projects (<https://github.com/openvinotoolkit/cvat/pull/3987>)
+- Added information to export CVAT_HOST when performing local installation for accessing over network (https://github.com/openvinotoolkit/cvat/pull/4014)
```
Comment (nmanovic) [inline CHANGELOG.md:39]:
```suggestion
- Added information to export CVAT_HOST when performing local installation for accessing over network (<https://github.com/openvinotoolkit/cvat/pull/4014>)
```

**Comment by anugrahasinha** [comment]
Thanks for merging the PR. 👍

**Pull request merged**